### PR TITLE
[SINGULAR] DDP-8422: Fix sticky section styles for mobile view

### PR DESCRIPTION
**Pull request by alextanlan**
The issue is described in the related [DDP-8422 ticket](https://broadinstitute.atlassian.net/browse/DDP-8422).

**_After the css fix:_**
mobile1

mobile2

mobile4

### CODE REVIEW

File: ddp-workspace/projects/ddp-dsm-ui/src/app/medical-record/medical-record.component.ts [hunk 1]
```diff
@@ -323,7 +323,7 @@ export class MedicalRecordComponent implements OnInit {
         const participantId = this.getParticipantId();
         const realm         = localStorage.getItem(ComponentService.MENU_SELECTED_REALM);
         const patch         = new PatchUtil(
-          this.medicalRecord.medicalRecordId,
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/medical-record/medical-record.component.ts]: fixed a linter warning.

File: ddp-workspace/projects/ddp-dsm-ui/src/app/tags/cohort-tag/bulk-cohort-tag-modal/bulk-cohort-tag-modal.component.ts [hunk 2]
```diff
@@ -44,6 +44,7 @@ export class BulkCohortTagModalComponent implements OnInit  {
     }
 
     // Clear the input value
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/tags/cohort-tag/bulk-cohort-tag-modal/bulk-cohort-tag-modal.component.ts:47]: fixed a linter warning.

File: ddp-workspace/projects/ddp-sdk/src/lib/services/configuration.service.ts [hunk 3]
```diff
@@ -124,5 +124,5 @@ export class ConfigurationService {
      * Configure additional fields to show for institutions
      */
     institutionsAdditionalFields: Partial<Record<InstitutionType, InstitutionAdditionalFields[]>> = {};
-    detail_MaxLength: number = 500;
+    detail_MaxLength = 500;
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-sdk/src/lib/services/configuration.service.ts:127]: fixed a linter warning.

File: ddp-workspace/projects/ddp-singular/src/app/components/activity/activity.component.scss [hunk 4]
```diff
@@ -37,61 +55,32 @@
...
 
-  @media (max-width: 1280px) {
-    top: 6rem;
+  @include media-lg {
+    top: $header-lg-height;
+    padding: 14px 24px;
+  }
+  @include media-sm {
+    top: $header-sm-height;
+    width: 100%;
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-singular/src/app/components/activity/activity.component.scss:72]: assuming we don't need a right margin for mobiles (see line 63)

**Review by juanseba21 — APPROVED** [review]

**Pull request merged**